utils/utils.go

```diff
@@ -59,3 +62,37 @@ func FuzzyAgo(ago time.Duration) string {
 
 	return fmtDuration(int(ago.Hours()/24/365), "year")
 }
+
+func GetTitle(cmd *cobra.Command, cmdType string, limit int, matchCount int, baseRepo *ghrepo.Interface) string {
```

Comment:
Since this is only shared between issues and PRs in the `command` package, I think this function should also live in an (unexported) function in the `command` package; no need to place it in `utils`. In fact, will probably minimize `utils` over time in favor of splitting things into tiny packages when it makes sense. For this though, I'm fine if it's under `command` for now.

Thank you for handling this!

Reply:
That makes sense, will move it to the command package